ktor-server/ktor-server-core/jvm/src/io/ktor/server/engine/DefaultTransformJvm.kt

```diff
@@ -33,16 +35,21 @@ internal actual suspend fun PipelineContext<Any, PipelineCall>.defaultPlatformTr
 @OptIn(InternalAPI::class)
 internal actual fun PipelineContext<*, PipelineCall>.multiPartData(rc: ByteReadChannel): MultiPartData {
     val contentType = call.request.header(HttpHeaders.ContentType)
-        ?: throw IllegalStateException("Content-Type header is required for multipart processing")
+        ?: throw UnsupportedMediaTypeException(null)
```

Comment:
There is also one more place, where exception is thrown - CIO: 
1)https://github.com/ktorio/ktor/blob/58d1714e5dadfe446430a492ac739c1058f84601/ktor-http/ktor-http-cio/jvm/src/io/ktor/http/cio/Multipart.kt#L141
2)https://github.com/ktorio/ktor/blob/58d1714e5dadfe446430a492ac739c1058f84601/ktor-http/ktor-http-cio/jvm/src/io/ktor/http/cio/Multipart.kt#L158
Can we please fix it too?